tests/Backup: Failures "Backup Framework: AlreadyExistsException: : Controls in framework ... already exist."

<!---
Please note the following potential times when an issue might be in Terraform core:

* [Configuration Language](https://www.terraform.io/docs/configuration/index.html) or resource ordering issues
* [State](https://www.terraform.io/docs/state/index.html) and [State Backend](https://www.terraform.io/docs/backends/index.html) issues
* [Provisioner](https://www.terraform.io/docs/provisioners/index.html) issues
* [Registry](https://registry.terraform.io/) issues
* Spans resources across multiple providers

If you are running into one of these scenarios, we recommend opening an issue in the [Terraform core repository](https://github.com/hashicorp/terraform/) instead.
--->

<!--- Please keep this note for the community --->

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

<!--- Thank you for keeping this note for the community --->

In nightly CI the Backup Framework acceptance tests are all failing with errors like:

```
=== RUN   TestAccBackupFramework_updateTags
=== PAUSE TestAccBackupFramework_updateTags
=== CONT  TestAccBackupFramework_updateTags
framework_test.go:85: Step 1/5 error: Error running apply: exit status 1
Error: error creating Backup Framework: AlreadyExistsException: Controls in framework tf_acc_test_q4nwp8c from account 123456789012 already exist.
{
RespMetadata: {
StatusCode: 400,
RequestID: "f1255d8f-1897-4a07-b0b6-ab37c923c790"
},
Message_: "Controls in framework tf_acc_test_q4nwp8c from account 123456789012 already exist."
}
with aws_backup_framework.test,
on terraform_plugin_test.tf line 2, in resource "aws_backup_framework" "test":
2: resource "aws_backup_framework" "test" {
--- FAIL: TestAccBackupFramework_updateTags (39.93s)
```

Relates https://github.com/hashicorp/terraform-provider-aws/pull/23193.
Relates https://github.com/hashicorp/terraform-provider-aws/pull/23175.

--- a/internal/service/backup/framework_data_source_test.go
+++ b/internal/service/backup/framework_data_source_test.go
@@ -11,12 +11,12 @@ import (
 	"github.com/hashicorp/terraform-provider-aws/internal/acctest"
 )
 
-func TestAccBackupFrameworkDataSource_basic(t *testing.T) {
+func testAccBackupFrameworkDataSource_basic(t *testing.T) {
 	datasourceName := "data.aws_backup_framework.test"
 	resourceName := "aws_backup_framework.test"
 	rName := fmt.Sprintf("tf_acc_test_%s", sdkacctest.RandString(7))
 
-	resource.ParallelTest(t, resource.TestCase{
+	resource.Test(t, resource.TestCase{
 		PreCheck:   func() { acctest.PreCheck(t); testAccFrameworkPreCheck(t) },
 		ErrorCheck: acctest.ErrorCheck(t, backup.EndpointsID),
 		Providers:  acctest.Providers,
@@ -67,12 +67,12 @@ func TestAccBackupFrameworkDataSource_basic(t *testing.T) {
 	})
 }
 
-func TestAccBackupFrameworkDataSource_controlScopeTag(t *testing.T) {
+func testAccBackupFrameworkDataSource_controlScopeTag(t *testing.T) {
 	datasourceName := "data.aws_backup_framework.test"
 	resourceName := "aws_backup_framework.test"
 	rName := fmt.Sprintf("tf_acc_test_%s", sdkacctest.RandString(7))
 
-	resource.ParallelTest(t, resource.TestCase{
+	resource.Test(t, resource.TestCase{
 		PreCheck:   func() { acctest.PreCheck(t); testAccFrameworkPreCheck(t) },
 		ErrorCheck: acctest.ErrorCheck(t, backup.EndpointsID),
 		Providers:  acctest.Providers,

--- a/internal/service/backup/framework_test.go
+++ b/internal/service/backup/framework_test.go
@@ -14,7 +14,36 @@ import (
 	tfbackup "github.com/hashicorp/terraform-provider-aws/internal/service/backup"
 )
 
-func TestAccBackupFramework_basic(t *testing.T) {
+func TestAccBackupFramework_serial(t *testing.T) {
+	testCases := map[string]map[string]func(t *testing.T){
+		"Resource": {
+			"basic":                        testAccBackupFramework_basic,
+			"disappears":                   testAccBackupFramework_disappears,
+			"UpdateTags":                   testAccBackupFramework_updateTags,
+			"UpdateControlScope":           testAccBackupFramework_updateControlScope,
+			"UpdateControlInputParameters": testAccBackupFramework_updateControlInputParameters,
+			"UpdateControls":               testAccBackupFramework_updateControls,
+		},
+		"DataSource": {
+			"basic":           testAccBackupFrameworkDataSource_basic,
+			"ControlScopeTag": testAccBackupFrameworkDataSource_controlScopeTag,
+		},
+	}
+
+	for group, m := range testCases {
+		m := m
+		t.Run(group, func(t *testing.T) {
+			for name, tc := range m {
+				tc := tc
+				t.Run(name, func(t *testing.T) {
+					tc(t)
+				})
+			}
+		})
+	}
+}
+
+func testAccBackupFramework_basic(t *testing.T) {
 	var framework backup.DescribeFrameworkOutput
 
 	rName := fmt.Sprintf("tf_acc_test_%s", sdkacctest.RandString(7))
@@ -22,7 +51,7 @@ func TestAccBackupFramework_basic(t *testing.T) {
 	updatedDescription := "updated description"
 	resourceName := "aws_backup_framework.test"
 
-	resource.ParallelTest(t, resource.TestCase{
+	resource.Test(t, resource.TestCase{
 		PreCheck:     func() { acctest.PreCheck(t); testAccFrameworkPreCheck(t) },
 		ErrorCheck:   acctest.ErrorCheck(t, backup.EndpointsID),
 		Providers:    acctest.Providers,
@@ -75,14 +104,14 @@ func TestAccBackupFramework_basic(t *testing.T) {
 	})
 }
 
-func TestAccBackupFramework_updateTags(t *testing.T) {
+func testAccBackupFramework_updateTags(t *testing.T) {
 	var framework backup.DescribeFrameworkOutput
 
 	rName := fmt.Sprintf("tf_acc_test_%s", sdkacctest.RandString(7))
 	description := "example description"
 	resourceName := "aws_backup_framework.test"
 
-	resource.ParallelTest(t, resource.TestCase{
+	resource.Test(t, resource.TestCase{
 		PreCheck:     func() { acctest.PreCheck(t); testAccFrameworkPreCheck(t) },
 		ErrorCheck:   acctest.ErrorCheck(t, backup.EndpointsID),
 		Providers:    acctest.Providers,
@@ -162,7 +191,7 @@ func TestAccBackupFramework_updateTags(t *testing.T) {
 	})
 }
 
-func TestAccBackupFramework_updateControlScope(t *testing.T) {
+func testAccBackupFramework_updateControlScope(t *testing.T) {
 	var framework backup.DescribeFrameworkOutput
 
 	rName := fmt.Sprintf("tf_acc_test_%s", sdkacctest.RandString(7))
@@ -171,7 +200,7 @@ func TestAccBackupFramework_updateControlScope(t *testing.T) {
 	updatedControlScopeTagValue := ""
 	resourceName := "aws_backup_framework.test"
 
-	resource.ParallelTest(t, resource.TestCase{
+	resource.Test(t, resource.TestCase{
 		PreCheck:     func() { acctest.PreCheck(t); testAccFrameworkPreCheck(t) },
 		ErrorCheck:   acctest.ErrorCheck(t, backup.EndpointsID),
 		Providers:    acctest.Providers,
@@ -274,7 +303,7 @@ func TestAccBackupFramework_updateControlScope(t *testing.T) {
 	})
 }
 
-func TestAccBackupFramework_updateControlInputParameters(t *testing.T) {
+func testAccBackupFramework_updateControlInputParameters(t *testing.T) {
 	var framework backup.DescribeFrameworkOutput
 
 	rName := fmt.Sprintf("tf_acc_test_%s", sdkacctest.RandString(7))
@@ -283,7 +312,7 @@ func TestAccBackupFramework_updateControlInputParameters(t *testing.T) {
 	updatedRequiredRetentionDays := "34"
 	resourceName := "aws_backup_framework.test"
 
-	resource.ParallelTest(t, resource.TestCase{
+	resource.Test(t, resource.TestCase{
 		PreCheck:     func() { acctest.PreCheck(t); testAccFrameworkPreCheck(t) },
 		ErrorCheck:   acctest.ErrorCheck(t, backup.EndpointsID),
 		Providers:    acctest.Providers,
@@ -356,14 +385,14 @@ func TestAccBackupFramework_updateControlInputParameters(t *testing.T) {
 	})
 }
 
-func TestAccBackupFramework_updateControls(t *testing.T) {
+func testAccBackupFramework_updateControls(t *testing.T) {
 	var framework backup.DescribeFrameworkOutput
 
 	rName := fmt.Sprintf("tf_acc_test_%s", sdkacctest.RandString(7))
 	description := "example description"
 	resourceName := "aws_backup_framework.test"
 
-	resource.ParallelTest(t, resource.TestCase{
+	resource.Test(t, resource.TestCase{
 		PreCheck:     func() { acctest.PreCheck(t); testAccFrameworkPreCheck(t) },
 		ErrorCheck:   acctest.ErrorCheck(t, backup.EndpointsID),
 		Providers:    acctest.Providers,
@@ -434,14 +463,14 @@ func TestAccBackupFramework_updateControls(t *testing.T) {
 	})
 }
 
-func TestAccBackupFramework_disappears(t *testing.T) {
+func testAccBackupFramework_disappears(t *testing.T) {
 	var framework backup.DescribeFrameworkOutput
 
 	rName := fmt.Sprintf("tf_acc_test_%s", sdkacctest.RandString(7))
 	description := "disappears"
 	resourceName := "aws_backup_framework.test"
 
-	resource.ParallelTest(t, resource.TestCase{
+	resource.Test(t, resource.TestCase{
 		PreCheck:     func() { acctest.PreCheck(t); testAccFrameworkPreCheck(t) },
 		ErrorCheck:   acctest.ErrorCheck(t, backup.EndpointsID),
 		Providers:    acctest.Providers,

